packages/@react-aria/landmark/test/useLandmark.test.tsx

```diff
@@ -0,0 +1,470 @@
+            <li><a href="/about">About</a></li>
+            <li><a href="/contact">Contact</a></li>
+          </ul>
+        </Navigation>
+        <Main>
+          <TextField label="First Name" />
+        </Main>
+      </div>
+    );
+
+    userEvent.tab();
+    userEvent.tab();
+    userEvent.tab();
+    userEvent.tab();
+    expect(document.activeElement).toBe(tree.getByRole('textbox'));
+
+    fireEvent.keyDown(document.activeElement, {key: 'F6'});
+    fireEvent.keyUp(document.activeElement, {key: 'F6'});
+    expect(document.activeElement).toBe(tree.getAllByRole('link')[2]);
+  });
+
+  it('components that already handle focus management, should handle focus themselves', function () {
+    let tree = render(
+      <Provider theme={theme}>
+        <div>
+          <Navigation>
+            <ActionGroup>
+              <Item>One</Item>
+              <Item>Two</Item>
+              <Item>Three</Item>
+            </ActionGroup>
+          </Navigation>
+          <Main>
+            <TableView aria-label="Table">
+              <TableHeader>
+                <Column key="foo" allowsSorting>Foo</Column>
+                <Column key="bar" allowsSorting>Bar</Column>
+                <Column key="baz">Baz</Column>
+              </TableHeader>
+              <TableBody>
+                <Row>
+                  <Cell>Foo 1</Cell>
+                  <Cell>Bar 1</Cell>
+                  <Cell>Baz 1</Cell>
+                </Row>
+                <Row>
+                  <Cell>Foo 2</Cell>
+                  <Cell>Bar 2</Cell>
+                  <Cell>Baz 2</Cell>
+                </Row>
+                <Row>
+                  <Cell>Foo 3</Cell>
+                  <Cell>Bar 3</Cell>
+                  <Cell>Baz 3</Cell>
+                </Row>
+              </TableBody>
+            </TableView>
+          </Main>
+        </div>
+      </Provider>
+    );
+    let buttons = tree.getAllByRole('button');
+    let table = tree.getByRole('grid');
+
+    userEvent.tab();
+    expect(document.activeElement).toBe(buttons[0]);
+    fireEvent.keyDown(document.activeElement, {key: 'ArrowRight'});
+    fireEvent.keyUp(document.activeElement, {key: 'ArrowRight'});
+    expect(document.activeElement).toBe(buttons[1]);
+
+    userEvent.tab();
+    expect(document.activeElement).toBe(table);
+    fireEvent.keyDown(document.activeElement, {key: 'ArrowDown'});
+    fireEvent.keyUp(document.activeElement, {key: 'ArrowDown'});
+    fireEvent.keyDown(document.activeElement, {key: 'ArrowRight'});
+    fireEvent.keyUp(document.activeElement, {key: 'ArrowRight'});
+    expect(document.activeElement).toBe(table);
+
+    fireEvent.keyDown(document.activeElement, {key: 'F6'});
+    fireEvent.keyUp(document.activeElement, {key: 'F6'});
+    expect(document.activeElement).toBe(buttons[1]);
+
+    fireEvent.keyDown(document.activeElement, {key: 'F6'});
+    fireEvent.keyUp(document.activeElement, {key: 'F6'});
+    expect(document.activeElement).toBe(table);
```

Comment:
this should be cell, mock the client height/width like table tests do

Reply:
Updated to do this